Bio/PDB/DSSP.py

```diff
@@ -267,27 +267,37 @@ class DSSP(AbstractResiduePropertyMap):
     -42.399999999999999)
     """
 
-    def __init__(self, model, pdb_file, dssp="dssp", acc_array="Sander"):
+    def __init__(self, model, in_file, dssp="dssp", acc_array="Sander", file_type='PDB'):
```

Comment:
I had to add this to the DSSP class to allow for using a pre-generated DSSP output file. If not adding it I don't know how to test the it without having to call dssp externally. The defaults are of course set to reproduce the old behaviour.